### PR TITLE
DOCS-9020 Update Beta Banners and Notes

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Update beta banner copy according to the Product Lifecycle Google Sheet. Whenever a beta banner is removed, we change the `is_beta` parameter to false.

- CCM
- Code Analysis
- Continuous Delivery
- Continuous Integration
- Quality Gates
- USM

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/cloud_cost_management/datadog_costs/_index.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ further_reading:
   text: "Quickly and comprehensively analyze the cloud and SaaS costs behind your services"
 ---
 
-{{< beta-callout url="https://www.datadoghq.com/private-beta/daily-datadog-costs/" >}}
-Daily Datadog costs in Cloud Cost Management are in private beta. To request access, complete the form.
-{{< /beta-callout >}}
+{{< callout url="https://www.datadoghq.com/private-beta/daily-datadog-costs/" btn_hidden="false" header="Join the Preview!">}}
```
Comment (alai97) [inline content/en/cloud_cost_management/datadog_costs/_index.md:30]:
```suggestion
{{< callout url="https://www.datadoghq.com/preview/daily-datadog-costs/" btn_hidden="false" header="Join the Preview!">}}
```

File: content/en/cloud_cost_management/multisource_querying/_index.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ further_reading:
   text: "Learn about Container Cost Allocation"
 ---
 
-{{< callout url="https://www.datadoghq.com/private-beta/multisource-querying/" btn_hidden="false" header="Join the Beta!">}}
-Multisource Querying is in private beta. To request access, complete the form.
+{{< callout url="https://www.datadoghq.com/private-beta/multisource-querying/" btn_hidden="false" header="Join the Preview!">}}
```
Comment (alai97) [inline content/en/cloud_cost_management/multisource_querying/_index.md:14]:
```suggestion
{{< callout url="https://www.datadoghq.com/preview/multisource-querying/" btn_hidden="false" header="Join the Preview!">}}
```

File: content/en/cloud_cost_management/custom.md [hunk 3]
```diff
@@ -18,9 +18,9 @@ further_reading:
   text: "Gain insights into your Google Cloud bill"
 ---
 
-{{< beta-callout url="#" btn_hidden="true" >}}
-Custom Cost is in public beta.
-{{< /beta-callout >}}
+{{< callout btn_hidden="true" header="Join the Preview!">}}
+Custom Costs are in preview.
```
Comment (rtrieu) [inline content/en/cloud_cost_management/custom.md:22]:
```suggestion
Custom Costs are in Preview.
```

File: content/en/cloud_cost_management/datadog_costs/_index.md [hunk 4]
```diff
@@ -27,9 +27,9 @@ further_reading:
...
+Daily Datadog costs in Cloud Cost Management are in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/cloud_cost_management/datadog_costs/_index.md:31]:
```suggestion
Daily Datadog costs in Cloud Cost Management are in Preview. If you're interested in this feature, complete the form to request access.
```

File: content/en/cloud_cost_management/multisource_querying/_index.md [hunk 5]
```diff
@@ -11,8 +11,8 @@ further_reading:
...
+Multisource Querying is in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/cloud_cost_management/multisource_querying/_index.md:15]:
```suggestion
Multisource Querying is in Preview. If you're interested in this feature, complete the form to request access.
```

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 6]
```diff
@@ -10,8 +10,8 @@ further_reading:
   text: "Learn about Cloud Cost Management"
 ---
 
-{{< callout url="#" btn_hidden="true" >}}
-Cloud Cost Recommendations is in public beta with support for AWS, and is automatically enabled if you have set up <a href="/cloud_cost_management/">Cloud Cost Management</a>.
+{{< callout btn_hidden="true" header="Join the Preview!">}}
+Cloud Cost Recommendations are in preview with support for AWS, and are automatically enabled if you have set up <a href="/cloud_cost_management/">Cloud Cost Management</a>.
```
Comment (rtrieu) [inline content/en/cloud_cost_management/recommendations/_index.md:14]:
```suggestion
Cloud Cost Recommendations are in Preview with support for AWS, and are automatically enabled if you have set up <a href="/cloud_cost_management/">Cloud Cost Management</a>.
```

File: content/en/cloud_cost_management/saas_costs.md [hunk 7]
```diff
@@ -24,9 +24,9 @@ further_reading:
   text: "Quickly and comprehensively analyze the cloud and SaaS costs behind your services"
 ---
 
-{{< beta-callout url="#" btn_hidden="true" >}}
-SaaS Cost Integrations are in public beta.
-{{< /beta-callout >}}
+{{< callout btn_hidden="true" header="Join the Preview!">}}
+SaaS Cost Integrations are in preview.
```
Comment (rtrieu) [inline content/en/cloud_cost_management/saas_costs.md:28]:
```suggestion
SaaS Cost Integrations are in Preview.
```

File: content/en/cloud_cost_management/tag_explorer/_index.md [hunk 8]
```diff
@@ -65,7 +65,7 @@ For Google Cloud tags, select **Google** from the dropdown menu on the top right
 {{% /tab %}}
 {{% tab "Datadog" %}}
 
-<div class="alert alert-warning">Daily Datadog costs are in private beta.</div>
+<div class="alert alert-warning">Daily Datadog costs are in preview.</div>
```
Comment (rtrieu) [inline content/en/cloud_cost_management/tag_explorer/_index.md:68]:
```suggestion
<div class="alert alert-warning">Daily Datadog costs are in Preview.</div>
```

File: content/en/cloud_cost_management/tag_explorer/_index.md [hunk 9]
```diff
@@ -74,7 +74,7 @@ For Datadog tags, select **Datadog** from the dropdown menu on the top right cor
 {{% /tab %}}
 {{% tab "Confluent Cloud" %}}
 
-<div class="alert alert-warning">Confluent Cloud costs are in public beta.</div>
+<div class="alert alert-warning">Confluent Cloud costs are in preview.</div>
```
Comment (rtrieu) [inline content/en/cloud_cost_management/tag_explorer/_index.md:77]:
```suggestion
<div class="alert alert-warning">Confluent Cloud costs are in Preview.</div>
```

File: content/en/cloud_cost_management/tag_explorer/_index.md [hunk 10]
```diff
@@ -83,7 +83,7 @@ For Confluent Cloud tags, select **Confluent Cloud** from the dropdown menu on t
 {{% /tab %}}
 {{% tab "Databricks" %}}
 
-<div class="alert alert-warning">Databricks costs are in public beta.</div>
+<div class="alert alert-warning">Databricks costs are in preview.</div>
```
Comment (rtrieu) [inline content/en/cloud_cost_management/tag_explorer/_index.md:86]:
```suggestion
<div class="alert alert-warning">Databricks costs are in Preview.</div>
```

File: content/en/cloud_cost_management/tag_explorer/_index.md [hunk 11]
```diff
@@ -92,7 +92,7 @@ For Databricks tags, select **Databricks** from the dropdown menu on the top rig
 {{% /tab %}}
 {{% tab "Fastly" %}}
 
-<div class="alert alert-warning">Fastly costs are in public beta.</div>
+<div class="alert alert-warning">Fastly costs are in preview.</div>
```
Comment (rtrieu) [inline content/en/cloud_cost_management/tag_explorer/_index.md:95]:
```suggestion
<div class="alert alert-warning">Fastly costs are in Preview.</div>
```

File: content/en/cloud_cost_management/tag_explorer/_index.md [hunk 12]
```diff
@@ -101,7 +101,7 @@ For Fastly tags, select **Fastly** from the dropdown menu on the top right corne
 {{% /tab %}}
 {{% tab "Elastic Cloud" %}}
 
-<div class="alert alert-warning">Elastic Cloud costs are in public beta.</div>
+<div class="alert alert-warning">Elastic Cloud costs are in preview.</div>
```
Comment (rtrieu) [inline content/en/cloud_cost_management/tag_explorer/_index.md:104]:
```suggestion
<div class="alert alert-warning">Elastic Cloud costs are in Preview.</div>
```

File: content/en/cloud_cost_management/tag_explorer/_index.md [hunk 13]
```diff
@@ -110,7 +110,7 @@ For Elastic Cloud tags, select **Elastic Cloud** from the dropdown menu on the t
 {{% /tab %}}
 {{% tab "MongoDB" %}}
 
-<div class="alert alert-warning">MongoDB costs are in public beta.</div>
+<div class="alert alert-warning">MongoDB costs are in preview.</div>
```
Comment (rtrieu) [inline content/en/cloud_cost_management/tag_explorer/_index.md:113]:
```suggestion
<div class="alert alert-warning">MongoDB costs are in Preview.</div>
```

File: content/en/cloud_cost_management/tag_explorer/_index.md [hunk 14]
```diff
@@ -119,7 +119,7 @@ For MongoDB tags, select **MongoDB** from the dropdown menu on the top right cor
 {{% /tab %}}
 {{% tab "OpenAI" %}}
 
-<div class="alert alert-warning">OpenAI costs are in public beta.</div>
+<div class="alert alert-warning">OpenAI costs are in preview.</div>
```
Comment (rtrieu) [inline content/en/cloud_cost_management/tag_explorer/_index.md:122]:
```suggestion
<div class="alert alert-warning">OpenAI costs are in Preview.</div>
```

File: content/en/cloud_cost_management/tag_explorer/_index.md [hunk 15]
```diff
@@ -128,7 +128,7 @@ For OpenAI tags, select **OpenAI** from the dropdown menu on the top right corne
 {{% /tab %}}
 {{% tab "Snowflake" %}}
 
-<div class="alert alert-warning">Snowflake costs are in public beta.</div>
+<div class="alert alert-warning">Snowflake costs are in preview.</div>
```
Comment (rtrieu) [inline content/en/cloud_cost_management/tag_explorer/_index.md:131]:
```suggestion
<div class="alert alert-warning">Snowflake costs are in Preview.</div>
```

File: content/en/cloud_cost_management/tag_explorer/_index.md [hunk 16]
```diff
@@ -137,7 +137,7 @@ For Snowflake tags, select **Snowflake** from the dropdown menu on the top right
 {{% /tab %}}
 {{% tab "Twilio" %}}
 
-<div class="alert alert-warning">Twilio costs are in public beta.</div>
+<div class="alert alert-warning">Twilio costs are in preview.</div>
```
Comment (rtrieu) [inline content/en/cloud_cost_management/tag_explorer/_index.md:140]:
```suggestion
<div class="alert alert-warning">Twilio costs are in Preview.</div>
```

File: content/en/code_analysis/static_analysis/_index.md [hunk 17]
```diff
@@ -106,7 +103,7 @@ The two types of fixes are distinguished visually in the UI with different label
 *AI Suggested Fixes:*
 {{< img src="code_analysis/static_analysis/static-analysis-ai-fix.png" alt="Visual indicator of an AI static analysis suggested fix" style="width:60%;">}}
 
-<div class="alert alert-warning">AI Suggested Fixes are in private beta. To request access, contact <a href="/help/">Support.</div>
+<div class="alert alert-warning">AI Suggested Fixes are in preview. To request access, contact <a href="/help/">Support.</div>
```
Comment (rtrieu) [inline content/en/code_analysis/static_analysis/_index.md:106]:
```suggestion
<div class="alert alert-warning">AI Suggested Fixes are in Preview. To request access, contact <a href="/help/">Support.</div>
```

File: content/en/continuous_delivery/_index.md [hunk 18]
```diff
@@ -27,12 +27,10 @@ cascade:
 <div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
-CD Visibility is in private beta. To request access, complete the form.
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" btn_hidden="false" header="Join the Preview!" >}}
+CD Visibility is in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/continuous_delivery/_index.md:31]:
```suggestion
CD Visibility is in Preview. If you're interested in this feature, complete the form to request access.
```

File: content/en/continuous_delivery/deployments/_index.md [hunk 19]
```diff
@@ -17,8 +17,8 @@ cascade:
 <div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
-CD Visibility is in private beta. To request access, complete the form.
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" btn_hidden="false" header="Join the Preview!" >}}
+CD Visibility is in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/continuous_delivery/deployments/_index.md:21]:
```suggestion
CD Visibility is in Preview. If you're interested in this feature, complete the form to request access.
```

File: content/en/continuous_delivery/deployments/argocd.md [hunk 20]
```diff
@@ -15,8 +15,8 @@ further_reading:
 <div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
-CD Visibility for Argo CD is in private beta. To request access, complete the form.
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" btn_hidden="false" header="Join the Preview!" >}}
+CD Visibility for Argo CD is in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/continuous_delivery/deployments/argocd.md:19]:
```suggestion
CD Visibility for Argo CD is in Preview. If you're interested in this feature, complete the form to request access.
```

File: content/en/continuous_delivery/deployments/ciproviders.md [hunk 21]
```diff
@@ -15,8 +15,8 @@ further_reading:
 <div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
-CD Visibility for CI providers deployments is in private beta. To request access, complete the form.
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" btn_hidden="false" header="Join the Preview!" >}}
+CD Visibility for CI provider deployments is in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/continuous_delivery/deployments/ciproviders.md:19]:
```suggestion
CD Visibility for CI provider deployments is in Preview. If you're interested in this feature, complete the form to request access.
```

File: content/en/continuous_delivery/explorer/_index.md [hunk 22]
```diff
@@ -14,8 +14,8 @@ further_reading:
 <div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
-CD Visibility is in private beta. To request access, complete the form.
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" btn_hidden="false" header="Join the Preview!" >}}
+CD Visibility is in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/continuous_delivery/explorer/_index.md:18]:
```suggestion
CD Visibility is in Preview. If you're interested in this feature, complete the form to request access.
```

File: content/en/continuous_delivery/explorer/facets.md [hunk 23]
```diff
@@ -14,8 +14,8 @@ further_reading:
 <div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
-CD Visibility is in private beta. To request access, complete the form.
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" btn_hidden="false" header="Join the Preview!" >}}
+CD Visibility is in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/continuous_delivery/explorer/facets.md:18]:
```suggestion
CD Visibility is in Preview. If you're interested in this feature, complete the form to request access.
```

File: content/en/continuous_delivery/explorer/saved_views.md [hunk 24]
```diff
@@ -11,8 +11,8 @@ further_reading:
 <div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
-CD Visibility is in private beta. To request access, complete the form.
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" btn_hidden="false" header="Join the Preview!" >}}
+CD Visibility is in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/continuous_delivery/explorer/saved_views.md:15]:
```suggestion
CD Visibility is in Preview. If you're interested in this feature, complete the form to request access.
```

File: content/en/continuous_delivery/explorer/search_syntax.md [hunk 25]
```diff
@@ -14,8 +14,8 @@ further_reading:
 <div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
-CD Visibility is in private beta. To request access, complete the form.
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" btn_hidden="false" header="Join the Preview!" >}}
+CD Visibility is in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/continuous_delivery/explorer/search_syntax.md:18]:
```suggestion
CD Visibility is in Preview. If you're interested in this feature, complete the form to request access.
```

File: content/en/continuous_delivery/search/_index.md [hunk 26]
```diff
@@ -11,8 +11,8 @@ further_reading:
 <div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
-CD Visibility is in private beta. To request access, complete the form.
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" btn_hidden="false" header="Join the Preview!" >}}
+CD Visibility is in preview. To request access, complete the form.
```
Comment (rtrieu) [inline content/en/continuous_delivery/search/_index.md:15]:
```suggestion
CD Visibility is in Preview. If you're interested in this feature, complete the form to request access.
```

**Review by goeln — APPROVED** [review]

**Review by brett0000FF — APPROVED** [review]

File: content/en/code_analysis/software_composition_analysis/_index.md [hunk 27]
```diff
@@ -22,16 +22,16 @@ algolia:
   tags: ['software composition analysis', 'datadog software composition analysis', 'library vulnerabilities', 'SCA']
 ---
 
+{{< callout url="#" btn_hidden="true" header="Join the Preview!" >}}
+Software Composition Analysis is in Preview.
```
Comment (rodrimendoza) [inline content/en/code_analysis/software_composition_analysis/_index.md:26]: We are changing it from `Code Analysis` to `Software Composition Analysis`. Is it fine?

**Review by rodrimendoza — APPROVED** [review]
LGTM

File: content/en/code_analysis/software_composition_analysis/_index.md (hunk 27 above)
Comment (rtrieu) [reply content/en/code_analysis/software_composition_analysis/_index.md:26]: Yes, that change isn't happening until 2025.

File: content/en/cloud_cost_management/multisource_querying/_index.md (hunk 2 above)
Comment (rtrieu) [inline content/en/cloud_cost_management/multisource_querying/_index.md:14]:
```suggestion
{{< callout url="https://www.datadoghq.com/product-preview/multisource-querying/" btn_hidden="false" header="Join the Preview!">}}
```

**Pull request merged**